rollup.config.js

```diff
@@ -1,10 +1,13 @@
-import typescript from "@rollup/plugin-typescript";
+import ts from "@wessberg/rollup-plugin-ts";
 import del from "rollup-plugin-delete";
 import pkg from "./package.json";
 
 const config = [
   {
-    external: Object.keys(pkg.peerDependencies || {}),
+    external: [
+      ...Object.keys(pkg.peerDependencies || {}),
+      "@material-ui/core/styles",
```

Comment:
this is needed because some imports in the library uses `import ... from '@material-ui/core/styles'` instead of what we have in the peerDependencies `@material-ui/core`. Rollup's warning is not smart enough to ignore subdirectories, thus needing to specifically add `@material-ui/core/styles` in the external list to avoid warnings